store/README.md

```diff
@@ -132,7 +132,6 @@ contains excessive notes and nullifiers, client can make additional filtering of
 * `chain_tip`: `uint32` – number of the latest block in the chain.
 * `block_header`: `BlockHeader` – block header of the block with the first note matching the specified criteria.
 * `mmr_delta`: `MmrDelta` – data needed to update the partial MMR from `block_num` to `block_header.block_num`.
```

Comment:
Should we change to:

`mmr_delta`: `MmrDelta` – data needed to update the partial MMR from `block_num + 1` to `block_header.block_num`.

And the propagate this change to other files?

Reply:
updated this file, which other files do you have in mind?

Reply:
`rpc/README.md`

Reply:
updated